bin/test-lxd-migration

```diff
@@ -146,4 +146,9 @@ for url in $(curl -s --unix-socket /var/snap/lxd/common/lxd/unix.socket "lxd/1.0
         echo "FAIL-IPv6: ${name}"
         FAIL=1
     fi
+
+    # Create some snapshots (catch possible error in sqlite_sequence)
+    lxc snapshot "${name}"
+    lxc snapshot "${name}"
+    lxc snapshot "${name}"
```

Comment:
We don't need to try it on all containers, so I'd do it after the loop and just test on `c1` or something.